ras-server.c

```diff
@@ -0,0 +1,363 @@
+#include <poll.h>
```

Comment:
Please add a license at the file. 

While here, please also add a Signed-off-by to this patch. 
(see https://www.kernel.org/doc/html/latest/process/submitting-patches.html?highlight=certificate%20origin#sign-your-work-the-developer-s-certificate-of-origin)